Project.toml

```diff
@@ -16,8 +17,7 @@ FiniteDifferences = "0.9"
 julia = "1"
 
 [extras]
-Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [targets]
-test = ["Random", "Test"]
+test = ["Test"]
```

Comment:
just noticed in passing - unrelated to this PR, but maybe nice to change - since `Test` is already in `deps` we don't need it here, and can remove `targets/extras` sections completely.